pkg/kubectl/cmd/version.go

```diff
@@ -37,6 +37,14 @@ type Version struct {
 	ServerVersion *apimachineryversion.Info `json:"serverVersion,omitempty" yaml:"serverVersion,omitempty"`
 }
 
+// VersionConfig: describe the options available to users of the "kubectl
+// version" command.
+type VersionConfig struct {
```

Comment:
I've updated this PR.

I've opened https://github.com/kubernetes/community/pull/795 to change `kubectl-conventions.md` in community. Thanks!